test/gerkin/features/data_create.feature

```diff
@@ -2,28 +2,18 @@
 Feature: API endpoint allowing users to create a new record in the Digital Registries database.
   Request endpoint: POST /data/{registryname}/{versionnumber}/create
 
-  Background:
-    Given The user wants to create a new record in the Digital Registries database
-
-  Scenario: The user successfully creates the record in the Digital Registries database
-    Given The record does not exist in the database 
-    When The user triggers an action to create a new record in the database
-    And The request with a valid payload is sent
-    Then Operation finishes successfully
-
-  Scenario: The user is not able to create a record that already exists in the Digital Registries database
-    Given The record already exists in the database 
-    When The user triggers an action to create a new record in the database
-    And The request with a valid payload is sent
-    Then Operation results in an error
+  Scenario: The user successfully creates the record "John Smith" in the Digital Registries database
+    Given The user wants to create a new record "John Smith" in the Digital Registries database
+    When The user triggers an action to create a new record "John Smith" in the database
+    Then Operation finishes successfully for create user "John Smith"
 
   Scenario: The user is not able to create a record in the database which not exist
+    Given The user wants to create a new record "Anna Stock" in the Digital Registries database
     Given The database does not exist
-    When The user triggers an action to create a new record in the database
-    And The request with a valid payload is sent
-    Then Operation results in an error
+    When The user triggers an action to create a new record "Anna Stock" in the database
+    Then Operation results in an error for create "Anna Stock"
 
   Scenario: The user is not able to create a record in the Digital Registries database
-    When The user triggers an action to create a new record in the database
-    And The request with an invalid payload is sent
-    Then Operation results in an error
+    Given The user wants to create a new record "Emma Watson" in the Digital Registries database
```

Comment:
Please avoid using celebrity names as tests data. 
Also this test case don't specify why given entry cannot be created.